controlplane/kubeadm/internal/webhooks/kubeadm_control_plane.go

```diff
@@ -70,26 +85,39 @@ func defaultKubeadmControlPlaneSpec(s *KubeadmControlPlaneSpec, namespace string
 		s.Version = "v" + s.Version
 	}
 
-	bootstrapv1.DefaultKubeadmConfigSpec(&s.KubeadmConfigSpec)
+	DefaultKubeadmConfigSpec(&s.KubeadmConfigSpec)
 
 	s.RolloutStrategy = defaultRolloutStrategy(s.RolloutStrategy)
 }
 
-func defaultRolloutStrategy(rolloutStrategy *RolloutStrategy) *RolloutStrategy {
+// DefaultKubeadmConfigSpec defaults a KubeadmConfigSpec.
+func DefaultKubeadmConfigSpec(r *bootstrapv1.KubeadmConfigSpec) {
```

Comment:
Please move DefaultKubeadmConfigSpec from bootstrap/kubeadm/internal/webhooks to bootstrap/kubeadm/api/v1beta1 and then use it here.

Reply:
Same for the usage in KubeadmControlPlaneTemplate.Default